src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -214,26 +214,21 @@ export const editVenueCastSchema = Yup.object()
   .from("config.landingPageConfig.bannerImageUrl", "bannerImageUrl")
 
   // possible locations for the map icon
-  .from("config.mapIconImageUrl", "mapIconImageUrl")
   .from("auditoriumColumns", "auditoriumColumns")
   .from("auditoriumRows", "auditoriumRows")
-  .from("mapIconImageUrl", "mapIconImageUrl")
   .from("code_of_conduct_questions", "code_of_conduct_questions")
   .from("profile_questions", "profile_questions");
 
 export const editPlacementCastSchema = Yup.object()
   .shape<Partial<PlacementInput>>({})
 
   // possible locations for the map icon
-  .from("config.mapIconImageUrl", "mapIconImageUrl")
-  .from("mapIconImageUrl", "mapIconImageUrl")
   .from("placement.addressText", "addressText")
   .from("placement.notes", "notes")
   .required();
 
 export const editPlacementSchema = Yup.object().shape<PlacementInput>({
   mapIconImageFile: createFileSchema("mapIconImageFile", false).notRequired(),
```

Comment:
Given we've removed `mapIconImageUrl`, would we not also be wanting to remove `mapIconImageFile`? Since presumably they are going to relate to the same sort of thing?